controllers/default.py

```diff
@@ -404,19 +409,25 @@ def contests():
     contests = []
     cal = pdt.Calendar()
 
-    table = TABLE(_class="centered striped")
+    button_class = "btn-floating btn-small accent-4 tooltipped"
+    view_link_class = button_class + " green view-contest"
+    reminder_class = button_class + " orange set-reminder"
+
+    table = TABLE(_class="centered striped", _id="contests-table")
     thead = THEAD(TR(TH("Contest Name"),
                      TH("Site"),
                      TH("Start"),
                      TH("Duration/Ending"),
                      TH("Link"),
-                     TH("Add Reminder")))
+                     TH("Add Reminder",
+                        BUTTON(I(_class="fa fa-refresh"),
```

Comment:
About the onLoad check (instead of refresh button): if the user browser doesn't accept third-party cookies, it will give "immediate failed" error, however I think this is acceptable.